e2e/integration_tests/keras_to_tfjs_create_save_predict.ts

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
It would be better to name this file `keras_to_tfjs_create_save_predict_test.ts` since it contains tests (we should also rename the existing `create_save_predict.ts` file to end with `tests.ts`).